packages/grafana-runtime/src/services/LocationService.ts

```diff
@@ -137,12 +137,13 @@ export function locationSearchToObject(search: string | number): UrlQueryMap {
 }
 
 /**
- * @alpha
+ * @public
  */
 export let locationService: LocationService = new HistoryWrapper();
 
-/** @internal
+/**
  * Used for tests only
+ * @internal
```

Comment:
Is this change only a code style change or did the previous position prevent it from being picked up by the api docs generate tooling?

Reply:
I need to verify but I think it didn't go picked up previously